podcasts/SwiftUI/SFSafariView.swift

```diff
@@ -12,3 +12,38 @@ struct SFSafariView: UIViewControllerRepresentable {
         // No need to do anything here
     }
 }
+
+struct SFSafariViewModifier: ViewModifier {
```

Comment:
While this looks like a good addition we have an Application Setting that controls if we want to open external links in the app or not. Not sure if we should check it in this modifier to decide the approach we want to use.
Check the code in `PodcastViewController` -> `open(url: URL)`

Reply:
Ah, that's a good catch.

I'm never sure what to do in the case of our "help" links for this setting. It seems like we typically use an in-app browser for things like Terms of Use and Privacy Policy, although not in the Upgrade screen:

| Upgrade Screen | Account Screen |
|--|--|
| <video src="https://github.com/user-attachments/assets/420d237e-7163-49f1-9d6a-39b9114e44fe"> | <video src="https://github.com/user-attachments/assets/62516646-b560-4c8d-93ec-31a9179f7ee3"> |

I think it would be nice to honor this setting everywhere, what do you think?

Reply:
Chatted with Dom on this and we think it's best to keep the internal "help" URLs as an in-app browser regardless of the user setting to keep them in the flow. External apps can be used for explicit "non-internal" links like podcasts or Show Notes links.